.github/workflows/flutter_package.yml

```diff
@@ -31,6 +31,10 @@ on:
         required: false
         type: string
         default: "80"
+      format_skip:
+        required: false
+        type: boolean
+        default: true
```

Comment:
Don't think we should be defaulting this to `true`. I understand needing the ability to skip the step but we purposely have this in our workflow to provide a consistent formatter to all code being submitted. This isn't anything we have had an issue with in the past.